ssl/record/methods/tls13_meth.c

```diff
@@ -78,27 +111,30 @@ static int tls13_cipher(OSSL_RECORD_LAYER *rl, TLS_RL_RECORD *recs,
 
     ctx = rl->enc_ctx;
     staticiv = rl->iv;
+    nonce = rl->nonce;
 
-    cipher = EVP_CIPHER_CTX_get0_cipher(ctx);
-    if (cipher == NULL) {
+    if (ctx == NULL && rl->mac_ctx == NULL) {
```

Comment:
I am not sure about the rename. It means that the code will diverge from older branches even further. On the other hand you're right that it would improve the clarity.

Reply:
If we keep names that do not (any more) fit, the code will become more and more hard to read and maintain,
which constitutes further legacy burden.

Reply:
Since it is local variable, I think renaming it for clarity will be better.